documents/TestSuite/sxpbrlib/multioutput/multioutput.mtlx

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<materialx version="1.36">
+  <nodegraph name="multioutput_test1">
```

Comment:
Simple tests to make sure multioutput nodes generate correctly, both when connected internally and when connected to graph output.